BraintreeCore/src/main/java/com/braintreepayments/api/AnalyticsClient.kt

```diff
@@ -217,6 +221,7 @@ internal class AnalyticsClient @VisibleForTesting constructor(
         const val WORK_INPUT_KEY_INTEGRATION = "integration"
         const val WORK_INPUT_KEY_SESSION_ID = "sessionId"
         const val WORK_INPUT_KEY_TIMESTAMP = "timestamp"
+        const val WORK_INPUT_KEY_PAYPAL_CONTEXT_ID = "paypal_context_id"
```

Comment:
Have we confirmed with the server SDK team that this is where they want this parameter passed in? I know at one point we discussed passing this in the `_meta` property to Arachne.

Reply:
Thanks, yeah, we will send `paypal_context_id` within `_meta `